pkg/grammar/SyslLexer.g4

```diff
@@ -191,7 +191,7 @@ TEXT_LINE       :
                 { !startsWithKeyword(ls(p), p.GetText()) }?
                 ;
 
-Name            : [a-zA-Z_][-a-zA-Z0-9_]*;
+Name            : [%a-zA-Z_][-%a-zA-Z0-9_]*;
```

Comment:
this regex isnt actually correct for url escaping, but its close enough I guess.... It really should be something like `(%[0-9a-fA-F][0-9a-fA-F])?`

Reply:
that would enforce it better, but what would the regex for `Name` be?